tests/cookbook/conftest.py

```diff
@@ -42,7 +42,10 @@ def service_requests_csv_pd_df():
     return pd.read_csv(COOKBOOK_DATA_CSV, keep_default_na=False)[COLUMNS]
 
 
-@pytest.fixture(scope="module", params=[1, 2])
+@pytest.fixture(
+    scope="module",
+    params=[1, 2] if daft.context.get_context().daft_execution_config.enable_native_executor is False else [1],
+)
```

Comment:
LGTM I think, but in the future is should really just be `if daft.context.get_context().runner == "ray"` since partitioning only makes sense for the ray runner